notify.plugin.zsh

```diff
@@ -10,7 +10,7 @@ zstyle ':notify:*' parent-pid $PPID
 # Notify an error with no regard to the time elapsed (but always only
 # when the terminal is in background).
 function notify-error {
-    notify-if-background error < /dev/stdin &!
```

Comment:
the redirection can be dropped as it's superfluous (I just put it as a sort of "reminder" that the function will use stdin), but `notify-if-background` should still be run in background, as all the AppleScript queries it runs can be sluggish sometimes (and the bang in `&!` will keep it from appearing in the output of `jobs`).